templates/base.html

```diff
@@ -32,7 +32,7 @@
 
     <footer class="container">
       <p>&copy; {{ now() | date(format="%Y") }} {{ config.title }}</p>
-      <p class="attribution"><a href="https://storyset.com" title="Storyset website">Site illustrations by Storyset</a></p>
+      <p class="attribution"><a href="https://storyset.com" title="Storyset website">Site illustrations by Storyset</a> | <a href="/contact/privacy" title="Limeleaf privacy policy">Privacy Policy</p>
```

Comment:
Using `{{ get_url(path="@/contact/privacy.md") }}` will safely generate the right URL for this page. https://www.getzola.org/documentation/templates/overview/#get-url

Examples of this is our main `<nav>` too.